test/integration/revocation_test.go

```diff
@@ -127,7 +127,7 @@ func TestPrecertificateRevocation(t *testing.T) {
 				tc.revokeClient.Account,
 				cert,
 				tc.revokeKey,
-				ocsp.KeyCompromise)
+				ocsp.Unspecified)
```

Comment:
Let's also make sure the integration test exercises the revocation path for reason = KeyCompromise. That probably means having a test case that's gated on BOULDER_CONFIG_DIR.